### PR TITLE
osu!mania Twins Mascot Design Contest newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
Goodbye, Maria. Don't get hit by a bus.

### CODE REVIEW

**Comment by cl8n** [comment]
😭

File: news/2019-09-06-a-most-manic-contest.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    * Each character can share similarities with the other, but they must be visually distinct and identifiable from one another.
+    * Characters must be humanoid. No anthropomorphic creatures.
+* All designs must be appropriate for an all-ages audience. Children do play this game, remember, and mascots are the "face" of their given gamemode.
+* Do not use features, themes or concepts from other intellectual properties. All entries must be unique and solely the creation of their authors. Plagiarisers will be exterminated with extreme prejudice.
+
+The top 3 entries as voted by the public will receive **6 months, 3 months, and 2 months** of osu!supporter respectively. 
+
+flyte will be reviewing all entries submitted himself and convening with whichever he deems the most fit to be selected as our new set of twin mascots — winning the contest does NOT mean that your entry will be chosen in that regard. But you've got a pretty good chance!
+
+Got all that? Great. [Head on over to the contest listing and submit your entry now!](https://osu.ppy.sh/community/contests/79) Entries will run for 30 days from the date of this post.
```
Comment (MegaApplePi) [inline news/2019-09-06-a-most-manic-contest.md]: temporary link? (It links to the Summer 2019 fanart voting page.)

File: news/2019-09-06-a-most-manic-contest.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+layout: post
+title: "A Most Manic Art Contest"
+date: 2019-09-06 04:00:00 +0000
+---
+
+Our dearly beloved osu!mania mascot Maria is packing up to head overseas for a music tour, and we need your help designing a new mascot to take her place. But here's the big catch: they're going to be **twins!**
+
+![](/wiki/shared/news/2019-09-06-a-most-manic-contest/banner.jpg)
+
+You heard me right, **twins** are the flavour of the month, and our latest art contest is featured on exactly that. Similar to the Yuzu's New Look contest from way back when, the intrepid artists among us have **30 days** to whip up concepts and designs for osu!mania's forthcoming twin mascots.
```
Comment (cl8n) [inline news/2019-09-06-a-most-manic-contest.md]:
```suggestion
You heard me right, **twins** are the flavour of the month, and our latest art contest is focused on exactly that. Similar to the *Yuzu's New Look* contest from way back when, the intrepid artists among us have **30 days** to whip up concepts and designs for osu!mania's forthcoming twin mascots.
```

File: news/2019-09-06-a-most-manic-contest.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+But why twins, you ask? Those unfamiliar with osu!mania may not realize that there are two major play styles for the mode in osu! - namely "four key" and "seven key" (4K and 7K) respectively. Maria may have been wonderful, but she's just one gal, and those were big shoes to fill.
```
Comment (cl8n) [inline news/2019-09-06-a-most-manic-contest.md]:
```suggestion
But why twins, you ask? Those unfamiliar with osu!mania may not realize that there are two major play styles for the mode in osu!—namely "four key" and "seven key" (4K and 7K) respectively. Maria may be wonderful, but she's just one gal, and these are big shoes to fill.
```

File: news/2019-09-06-a-most-manic-contest.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* The chief theme for this contest is **twins**.
+* Your entry must contain two designs for two characters.
+    * Each character can share similarities with the other, but they must be visually distinct and identifiable from one another.
+    * Characters must be humanoid. No anthropomorphic creatures.
+* All designs must be appropriate for an all-ages audience. Children do play this game, remember, and mascots are the "face" of their given gamemode.
+* Do not use features, themes or concepts from other intellectual properties. All entries must be unique and solely the creation of their authors. Plagiarisers will be exterminated with extreme prejudice.
+
+The top 3 entries as voted by the public will receive **6 months, 3 months, and 2 months** of osu!supporter respectively. 
+
+flyte will be reviewing all entries submitted himself and convening with whichever he deems the most fit to be selected as our new set of twin mascots — winning the contest does NOT mean that your entry will be chosen in that regard. But you've got a pretty good chance!
```
Comment (cl8n) [inline news/2019-09-06-a-most-manic-contest.md]:
```suggestion
flyte will be reviewing all entries submitted himself and convening with whichever he deems the most fit to be selected as our new set of twin mascots—winning the contest does *NOT* mean that your entry will be chosen in that regard, but you've got a pretty good chance!
```

**Comment by cl8n** [comment]
should remove banner@2x as well, looks like it's just upscaled from the original

**Pull request merged**